ossdev/__init__.py

```diff
@@ -34,38 +35,50 @@ def __hash__(self):
         return sum(self.d)
 
     def __setitem__(self, key, value):
-        # TODO: implement
-        return None
+        self.d[key] = value
 
     def __cmp__(self, other):
-        # TODO: implement, -1 if self < other, 0 if self == other, 1 if self > other
-        return -1
+        return len(self) - len(other)
+
+    def __eq__(self, other):
```

Comment:
Implementing `cmp` should suffice, implementations of other comparators seems redundant.